readme.md

```diff
@@ -3,30 +3,47 @@
 # AMP Service worker
 AMP service worker is a service worker library for your AMP pages.
 
-The project aims to bring network resiliency and other network related optimizations to any AMP page which is being controlled by this service worker.
-Core use cases:
+**Purpose:** The project aims to bring network resiliency and other network related optimizations to any AMP page which is being controlled by this service worker.
```

Comment:
Perhaps...

Purpose: Enhance network resiliency and optimize AMP asset reuse for AMP documents.